.github/workflows/sycl-nightly.yml

```diff
@@ -107,6 +107,22 @@ jobs:
       runner: '["Windows","gen12"]'
       sycl_toolchain_archive: ${{ needs.build-win.outputs.artifact_archive_name }}
 
+  sycl-cts:
```

Comment:
I think we can actually merge that into the e2e matrix.

Reply:
[441647a](https://github.com/intel/llvm/pull/12811/commits/441647aa10085f0ba6b4be8eb93359f7f844da05)
Not sure what will be `matrix.tests_selector` for the rest